app/models/setting.rb

```diff
@@ -10,15 +10,17 @@ class Setting < ActiveRecord::Base
             :event_end_date,
             :event_url,
             :available_spots,
+            :required_rates_num,
+            :confirmation_time,
             presence: true
-  validates :available_spots, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-  validates :required_rates_num, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-
+  validates :available_spots, :required_rates_num, :confirmation_time,
+            numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
   def self.get
     self.first || self.create({
       available_spots: 0,
       required_rates_num: 3,
+      confirmation_time: 7,
```

Comment:
Regarding the comment about naming, an idea popped up in my mind: days_to_confirm_invitation. Although, I'm not sure about it.